allennlp/data/dataset_readers/wikitables/wikitables_dataset_reader.py

```diff
@@ -64,18 +65,36 @@ class WikiTablesDatasetReader(DatasetReader):
         Token indexers for table entities. Will default to ``question_token_indexers`` (though you
         very likely want to use something different for these, as you can't rely on having an
         embedding for every table entity at test time).
+    nonterminal_indexers : ``Dict[str, TokenIndexer]`` (optional)
+        How should we represent non-terminals in production rules when we're computing action
+        embeddings?  We use ``TokenIndexers`` for this.  Default is to use a
+        ``SingleIdTokenIndexer`` with the ``rule_labels`` namespace, keyed by ``tokens``:
+        ``{"tokens": SingleIdTokenIndexer("rule_labels")}``.  We use the namespace ``rule_labels``
+        so that we don't get padding or OOV tokens for nonterminals.
+    terminal_indexers : ``Dict[str, TokenIndexer]`` (optional)
+        How should we represent terminals in production rules when we're computing action
+        embeddings?  We also use ``TokenIndexers`` for this.  The default is to use a
+        ``TokenCharactersIndexer`` keyed by ``token_characters``: ``{"token_characters":
+        TokenCharactersIndexer()}``.  We use this indexer by default because WikiTables has plenty
```

Comment:
Because of how the indexers work, I'm not actually sure that's a good idea.  The way that type embeddings are used in the original parser is different from how it would work if we included the type as a token indexer here.  Maybe that's a good idea for something to try later, after we've replicated the original model, but it would be a different model.